index.html

```diff
@@ -129,9 +131,61 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
         <div class="container">
 
         <!-- mainline releases v4.8.0.5 -->
- 
+
 <!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
 
+<div class="smaller-margin" id="mr-16">
+        <h2 class="text-middle">Mainline releases</h2>
+        <!-- h2 tag needs to be moved to the newest version -->
+        <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.8.0.5...v4.8.0.6">v4.8.0.6</a></h4>
+    </div>
+
+    <div class="dimmed-text"><p>Release date: 2018-11-08</p></div>
+
+    <ul>
+        <li>Fixed/added:</li>
+        <ol>
+          <li>SeaBIOS: updated to rel-1.11.0.6</li>
+          <li>coreboot: added console output redirection to COM2 option</li>
+          <li>coreboot: synced and rebased with coreboot official repo commit de46280</li>
+        </ol>
+        <li>Known issues:</li>
+        <ol>
+            <li><a href="https://github.com/pcengines/coreboot/issues/196">CPU frequency is stuck at low frequencies and does not react to stressin"</a></li>
+            <li><a href="https://github.com/pcengines/apu2-documentation/issues/115">some PCIe cards are not detected on certain OSes</a></li>
+            <li><a href="https://github.com/pcengines/seabios/issues/30">booting with 2 USB 3.x sticks plugged in apu4 sometimes results in detecting only 1 stick</a></li>
+            <li><a href="https://github.com/pcengines/seabios/issues/29">certain USB 3.x sticks happen to not appear in boot menu</a></li>
+            <li><a href="https://github.com/pcengines/apu2-documentation/issues/109">booting Xen is unstable</a></li>
+            <li><a href="https://github.com/pcengines/apu2-documentation/issues/64">platforms happen to hang after reboot</a></li>
+        </ol>
+    </ul>
+
+    <div class="source-code-links-binaries">
+        <div class="source-code-column">
+            <div class="smaller-margin"><p>Binaries:</p></div>
+            <ul>
+                <li><a href="http://pcengines.ch/file/apu1_v4.8.0.6.rom.tar.gz">apu1 v4.8.0.6/a></li>
+                <li><a href="http://pcengines.ch/file/apu2_v4.8.0.6.rom.tar.gz">apu2 v4.8.0.6</a></li>
+                <li><a href="http://pcengines.ch/file/apu3_v4.8.0.6.rom.tar.gz">apu3 v4.8.0.6</a></li>
+                <li><a href="http://pcengines.ch/file/apu4_v4.8.0.6.rom.tar.gz">apu4 v4.8.0.6</a></li>
+                <li><a href="http://pcengines.ch/file/apu5_v4.8.0.6.rom.tar.gz">apu5 v4.8.0.6</a></li>
+            </ul>
+        </div>
+        <div class="source-code-column">
+            <div class="smaller-margin"><p>Source code:</p></div>
+            <ul>
+                <li><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.8.0.5...v4.8.0.6">coreboot v4.8.0.5</a></li>
```

Comment:
@moody92 the displayed text is still coreboot v4.8.0.5

Reply:
done